packages/protobuf-test/src/gen/ts/extra/msg-json-names_pb.ts

```diff
@@ -72,7 +72,7 @@ export class JsonNamesMessage extends Message<JsonNamesMessage> {
     { no: 4, name: "b", jsonName: "sameJsonName", kind: "scalar", T: 9 /* ScalarType.STRING */ },
     { no: 5, name: "c", kind: "scalar", T: 9 /* ScalarType.STRING */ },
     { no: 6, name: "d", jsonName: "c", kind: "scalar", T: 9 /* ScalarType.STRING */ },
-    { no: 7, name: "e", kind: "scalar", T: 9 /* ScalarType.STRING */ },
+    { no: 7, name: "e", jsonName: "", kind: "scalar", T: 9 /* ScalarType.STRING */ },
```

Comment:
The most irrelevant bugfix? Yes, protoc allows this, and doesn't care about name conflicts in `json_name` either.